code/Location.php

```diff
@@ -88,34 +87,23 @@ function fieldLabels($includerelations = true)
 
     public function getCMSFields()
     {
-
         $fields = parent::getCMSFields();
```

Comment:
Per our conversation earlier, do we want to move this away from `parent::getCMSFields()` since it's a `DataObject`?